pyproject.toml

```diff
@@ -7,13 +10,28 @@ skips = ["*/*test.py", "*/test_*.py", "*tests/*.py"]
 [tool.coverage.run]
 branch = true
 
+[tool.coverage.report]
+# fail_under = 99
+show_missing = true
+
+[tool.pytest.ini_options]
+minversion = "6.0"
+log_cli_level = "INFO"
+
+[tool.pylint]
+disable = "fixme"
```

Comment:
Why this?

Reply:
I added this to disable lint warnings on `TODO:` comments, some are out of scope of the minimal charm. I'll remove this line once I've addressed all of them